context/TransactionContext/index.tsx

```diff
@@ -75,23 +98,41 @@ export const TransactionStore: React.FC = ({ children }: Children) => {
 
             updateToast(toastId as unknown as string, {
                 content: [
-                    statusMessages?.success?.title ?? 'Transaction Successful',
+                    statusMessages?.success?.title ?? 'Order Submitted to Queue',
                     statusMessages?.success?.body ?? (
-                        <a
-                            key={contractReceipt.transactionHash}
-                            href={`${networkConfig[network ?? ARBITRUM]?.previewUrl}/${
-                                contractReceipt.transactionHash
-                            }`}
-                            className="text-tracer-400 underline"
-                            target="_blank"
-                            rel="noreferrer"
-                        >
-                            View transaction
-                        </a>
+                        <>
+                            <div className="mb-2">
+                                {`${statusMessages?.poolName} in `}
+                                <TimeLeft targetTime={statusMessages?.nextRebalance ?? 0} />
```

Comment:
This will change for all transactions that dont have a body. Its probably better if its just put into the body when committing

Reply:
Good point, added a check to see if there is a `nextBalance` being passed in